docs/development/extensions-core/kafka-supervisor-reference.md

```diff
@@ -37,7 +37,7 @@ This topic contains configuration reference information for the Apache Kafka sup
 
 |Field|Type|Description|Required|
 |-----|----|-----------|--------|
-|`topic`|String|The Kafka topic to read from. Must be a specific topic. Topic patterns are not supported.|yes|
+|`topic`|String|The Kafka topic to read from. Must be a specific topic. Topic patterns are supported if `multiTopic` is enabled.|yes|
```

Comment:
I think this should be restructured slightly to a form like this:

```
If `multiTopic` is not set (or false) then `topic` is treated as a string that represents the Kafka topic from which the data will be read.
If `multiTopic` is true then `topic` represents a regular expression that can match (exactly) several topics.
```

One of the things I am not clear on (and want to see documentation for) is the `exactly` part.
Is the topic regex anchored in the start and end?
If I have topics like `metrics-stage`, `metrics-dev`, `metrics-prod` if multiTopic is true should topic be set to `metrics-` or `metrics-.*`  ?

Reply:
rephrasing it a bit more 
The `topic` is interpreted differently depending on whether the multi-topic ingestion (link-to-anchor for multi-topic) is enabled or not. If multi-topic ingestion is enabled, `topic` represents a regular expression that can match several topics. Check (link-to-anchor) for more details. If multi-topic ingestion is not enabled, topic is considered a single kafka topic to read from.

Reply:
I have made the above changes.